lib/src/collision/broadphase/pair.dart

```diff
@@ -38,7 +14,9 @@ class Pair implements Comparable<Pair> {
     if (this.proxyIdA == pair2.proxyIdA) {
       return proxyIdB < pair2.proxyIdB
           ? -1
-          : proxyIdB == pair2.proxyIdB ? 0 : 1;
+          : proxyIdB == pair2.proxyIdB
+              ? 0
+              : 1;
```

Comment:
I would add a parenthesis on the internal ternary and inline it back

Reply:
well actually is this a poor means compareTo? I am sure dart has some built in for this

Reply:
Yeah, this is just code that looks changed because it has been auto-formatted, but I'll look into changing these ones.